package/MDAnalysis/analysis/nuclinfo.py

```diff
@@ -146,24 +153,31 @@ def wc_pair(universe, i, bp, seg1="SYSTEM", seg2="SYSTEM"):
 
 
 def minor_pair(universe, i, bp, seg1="SYSTEM", seg2="SYSTEM"):
-    """Minor-Groove basepair distance for residue *i* with residue *bp*.
+    """Minor-Groove basepair distance for residue `i` with residue `bp`.
 
     The distance of the nitrogen and oxygen atoms in a Minor-groove hydrogen
     bond is computed.
 
-    :Arguments:
-      *universe*
-          :class:`~MDAnalysis.core.universe.Universe` containing the trajectory
-      *seg1*
-          segment id for first base
-      *i*
-          resid of the first base
-      *seg2*
-          segment id for second base
-      *bp*
-          resid of the second base
+    Parameters
+    ----------
+    universe : Universe
+         :class:`~MDAnalysis.core.universe.Universe` containing the trajectory
+    i : int
+        resid of the first base
+    bp : int
+        resid of the second base
+    seg1 : str (optional)
+        segment id for first base ["SYSTEM"]
+    seg2 : str (optional)
+        segment id for second base ["SYSTEM"]
+
+    Returns
+    -------
+    float
+        Minor groove base pair distance
 
-    .. NOTE:: If failure occurs be sure to check the segment identification.
+
+    .. note:: If failure occurs be sure to check the segment identification.
```

Comment:
Changed to Notes section (rebased so it now shows up in commit 056f0976100e02c82a354a76be79f0669549926e )